projects/demo/src/modules/components/combo-box/combo-box.component.ts

```diff
@@ -115,8 +115,9 @@ export class ExampleTuiComboBoxComponent extends AbstractExampleTuiControl {
 
     stringify = this.stringifyVariants[0];
 
-    readonly strictMatcherVariants: ReadonlyArray<TuiStringMatcher<Account>> = [
+    readonly strictMatcherVariants: ReadonlyArray<TuiStringMatcher<Account> | null> = [
         TUI_STRICT_MATCHER as TuiStringMatcher<Account>,
+        null,
```

Comment:
We don't need it explicitly in the list, null in type would enable cleaner on the docs